addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -318,7 +359,7 @@ MockServer.include({
             return Promise.resolve(this._mockChannelFetchPreview(args));
         }
         if (args.method === 'channel_fold') {
-            return;
+            return Promise.resolve(this._mockChannelFold(args));
```

Comment:
method is async, it can just return the value without wrapping a Promise

Reply:
It's fine for consistency, better make a full-pass I would say. Not super important regardless of decision in this PR though...